src/OpenFeature/ProviderRepository.cs

```diff
@@ -242,11 +206,9 @@ await InitProviderAsync(this._defaultProvider, context, afterInitialization, aft
             try
             {
                 await targetProvider.ShutdownAsync().ConfigureAwait(false);
-                afterShutdown?.Invoke(targetProvider);
             }
-            catch (Exception ex)
+            catch (Exception)
```

Comment:
Is there any reason why we decided to remove the exception here?

Reply:
It's now unused. We before passed an optional lambda that wasn't ever being set, except in tests.

I've added some logging here instead.